public/index.html

```diff
@@ -6,23 +6,69 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="description" content="">
     <meta name="author" content="Virtual Video Game Orchestra">
-    <meta name="generator" content="Jekyll v3.8.6">
 
     <title>Virtual Video Game Orchestra</title>
 
     <link rel="icon" href="images/favicons/favicon-32x32.png" sizes="32x32" type="image/png">
+    <link rel="stylesheet" type="text/css" href="npm/bootstrap/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="npm/@fortawesome/fontawesome-free/css/all.min.css">
     <link rel="stylesheet" type="text/css" href="css/theme.css">
+    <link rel="stylesheet" type="text/css" href="css/crt-tv.css">
+
+    <script type="text/javascript" src="npm/jquery/dist/jquery.min.js"></script>
+    <script type="text/javascript" src="npm/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="npm/@fortawesome/fontawesome-free/js/fontawesome.min.js"></script>
 </head>
 
+<div class="container">
+    <nav class="top-nav navbar navbar-expand-md navbar-dark bg-dark-blue-transparent fa-border m-2">
+        <a class="nav-link btn-outline-primary text-light" href="/">
+            <i class="fas fa-home fa-2x mr-2"></i>Virtual VGO
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse"
+                aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarCollapse">
+            <ul class="navbar-nav container">
+                <li class="nav-item btn">
+                    <a class="nav-link btn-outline-primary text-monospace btn-link" href="/parts">
+                        Parts
+                    </a>
+                </li>
+            </ul>
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link btn-outline-primary" href="https://www.facebook.com/groups/1080154885682377/">
+                        <i class="fab fa-facebook-square fa-2x"></i>
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link btn-outline-primary" href="https://github.com/virtual-vgo/vvgo">
+                        <i class="fab fa-github-square fa-2x"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+</div>
+
 <body>
-<div><img src="images/logo.svg" alt="logo" class="logo"></div>
-<div>
-    <table class="button-table">
-        <tr>
-            <td><a href="https://www.facebook.com/groups/1080154885682377/" class="button facebook">Facebook Group</a>
-            </td>
-        </tr>
-    </table>
+<div class="container jumbotron bg-transparent">
+    <div class="bg-terminal">
```

Comment:
Remove the `.bg-terminal` div, and add `.bg-terminal` to the outer.

The rest of the Jumbotron issues, I'll deal with as part of the designated papercut issue.

Reply:
I need the outer to have a transparent background otherwise it doesn't look right. I'm not sure how to do it without the three layers.

Reply:
I'll take a look at it later, then.

Reply:
Later being "not this PR".